app.py

```diff
@@ -0,0 +1,48 @@
+from flask import Flask, request, jsonify
+from flask_sqlalchemy import SQLAlchemy
+from models import db, Profile
+
+
+app = Flask(__name__)
+
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///todos.db'
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+db.init_app(app)
+
+
+@app.route('/register', methods=['POST'])
+def add_user():
+    data = request.json
+    email_id = data.get('email_id')
+    username = data.get('username')
+    password = data.get('password')
```

Comment:
To store the password in an encrypted format rather than plain text in the database to ensure better security.